### ISSUE TITLE
Windows app will not compile - flutter 2.10.1

### ISSUE THREAD

**Issue by brianoh** [flutter/flutter#98664]
## Steps to Reproduce

1. Using Latest Flutter 2.10.1. Create a new app.
2. Create a Windows version.
3. Attempt to run the Windows version. This fails.

## Logs

<details>
<summary>
Logs
</summary>

```
c:\Users\brian2\flutter-dev\testwin01\win10>flutter --verbose run
[  +54 ms] executing: [c:\Users\brian2\flutter/] git -c log.showSignature=false log -n 1 --pretty=format:%H
[  +58 ms] Exit code 0 from: git -c log.showSignature=false log -n 1 --pretty=format:%H
[        ] db747aa1331bd95bc9b3874c842261ca2d302cd5
[        ] executing: [c:\Users\brian2\flutter/] git tag --points-at db747aa1331bd95bc9b3874c842261ca2d302cd5
[  +84 ms] Exit code 0 from: git tag --points-at db747aa1331bd95bc9b3874c842261ca2d302cd5
[        ] 2.10.1
[   +6 ms] executing: [c:\Users\brian2\flutter/] git rev-parse --abbrev-ref --symbolic @{u}
[  +21 ms] Exit code 0 from: git rev-parse --abbrev-ref --symbolic @{u}
[        ] origin/stable
[        ] executing: [c:\Users\brian2\flutter/] git ls-remote --get-url origin
[  +17 ms] Exit code 0 from: git ls-remote --get-url origin
[        ] https://github.com/flutter/flutter.git
[  +73 ms] executing: [c:\Users\brian2\flutter/] git rev-parse --abbrev-ref HEAD
[  +23 ms] Exit code 0 from: git rev-parse --abbrev-ref HEAD
[        ] stable
[  +41 ms] Artifact Instance of 'AndroidGenSnapshotArtifacts' is not required, skipping update.
[   +1 ms] Artifact Instance of 'AndroidInternalBuildArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'IOSEngineArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'FlutterWebSdk' is not required, skipping update.
[   +3 ms] Artifact Instance of 'WindowsEngineArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'WindowsUwpEngineArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'MacOSEngineArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'LinuxEngineArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'LinuxFuchsiaSDKArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'MacOSFuchsiaSDKArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'FlutterRunnerSDKArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'FlutterRunnerDebugSymbols' is not required, skipping update.
[  +41 ms] executing: C:\Users\brian2\AppData\Local\Android\sdk\platform-tools\adb.exe devices -l
[  +34 ms] List of devices attached
[   +5 ms] Artifact Instance of 'AndroidGenSnapshotArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'AndroidInternalBuildArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'IOSEngineArtifacts' is not required, skipping update.
[   +3 ms] Artifact Instance of 'WindowsUwpEngineArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'MacOSEngineArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'LinuxEngineArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'LinuxFuchsiaSDKArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'MacOSFuchsiaSDKArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'FlutterRunnerSDKArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'FlutterRunnerDebugSymbols' is not required, skipping update.
[  +63 ms] Skipping pub get: version match.
[ +112 ms] Initializing file store
[  +10 ms] Skipping target: gen_localizations
[   +5 ms] gen_dart_plugin_registrant: Starting due to {InvalidatedReasonKind.inputChanged: The following inputs have updated contents:
C:\Users\brian2\flutter-dev\testwin01\win10\.dart_tool\package_config_subset}
[  +24 ms] gen_dart_plugin_registrant: Complete
[   +1 ms] Skipping target: _composite
[   +1 ms] complete
[   +4 ms] Launching lib\main.dart on Windows in debug mode...
[   +4 ms] c:\Users\brian2\flutter\bin\cache\dart-sdk\bin\dart.exe --disable-dart-dev c:\Users\brian2\flutter\bin\cache\artifacts\engine\windows-x64\frontend_server.dart.snapshot --sdk-root
c:\Users\brian2\flutter\bin\cache\artifacts\engine\common\flutter_patched_sdk/ --incremental --target=flutter --debugger-module-names --experimental-emit-debug-metadata
-DFLUTTER_WEB_AUTO_DETECT=true --output-dill C:\Users\brian2\AppData\Local\Temp\flutter_tools.9a3dd30b\flutter_tool.202a0d43\app.dill --packages
c:\Users\brian2\flutter-dev\testwin01\win10\.dart_tool\package_config.json -Ddart.vm.profile=false -Ddart.vm.product=false --enable-asserts --track-widget-creation --filesystem-scheme
org-dartlang-root --initialize-from-dill build\c075001b96339384a97db4862b8ab8db.cache.dill.track.dill --enable-experiment=alternative-invalidation-strategy
[  +20 ms] executing: C:\Program Files (x86)\Microsoft Visual Studio\Installer\vswhere.exe -format json -products * -utf8 -latest -version 16 -requires
Microsoft.VisualStudio.Workload.NativeDesktop Microsoft.VisualStudio.Component.VC.Tools.x86.x64 Microsoft.VisualStudio.Component.VC.CMake.Project
[  +29 ms] Exit code 0 from: C:\Program Files (x86)\Microsoft Visual Studio\Installer\vswhere.exe -format json -products * -utf8 -latest -version 16 -requires
Microsoft.VisualStudio.Workload.NativeDesktop Microsoft.VisualStudio.Component.VC.Tools.x86.x64 Microsoft.VisualStudio.Component.VC.CMake.Project
[   +2 ms] [
             {
               "instanceId": "c6ed6aa4",
               "installDate": "2022-02-17T08:15:18Z",
               "installationName": "VisualStudio/17.1.0+32210.238",
               "installationPath": "C:\\Program Files\\Microsoft Visual Studio\\2022\\Community",
               "installationVersion": "17.1.32210.238",
               "productId": "Microsoft.VisualStudio.Product.Community",
               "productPath": "C:\\Program Files\\Microsoft Visual Studio\\2022\\Community\\Common7\\IDE\\devenv.exe",
               "state": 4294967295,
               "isComplete": true,
               "isLaunchable": true,
               "isPrerelease": false,
               "isRebootRequired": false,
               "displayName": "Visual Studio Community 2022",
               "description": "Powerful IDE, free for students, open-source contributors, and individuals",
               "channelId": "VisualStudio.17.Release",
               "channelUri": "https://aka.ms/vs/17/release/channel",
               "enginePath": "C:\\Program Files (x86)\\Microsoft Visual Studio\\Installer\\resources\\app\\ServiceHub\\Services\\Microsoft.VisualStudio.Setup.Service",
               "installedChannelUri": "https://aka.ms/vs/17/release/channel",
               "releaseNotes": "https://docs.microsoft.com/en-us/visualstudio/releases/2022/release-notes-v17.1#17.1.0",
               "thirdPartyNotices": "https://go.microsoft.com/fwlink/?LinkId=661288",
               "updateDate": "2022-02-17T08:15:18.0297586Z",
               "catalog": {
                 "buildBranch": "d17.1",
                 "buildVersion": "17.1.32210.238",
                 "id": "VisualStudio/17.1.0+32210.238",
                 "localBuild": "build-lab",
                 "manifestName": "VisualStudio",
                 "manifestType": "installer",
                 "productDisplayVersion": "17.1.0",
                 "productLine": "Dev17",
                 "productLineVersion": "2022",
                 "productMilestone": "RTW",
                 "productMilestoneIsPreRelease": "False",
                 "productName": "Visual Studio",
                 "productPatchVersion": "0",
                 "productPreReleaseMilestoneSuffix": "7.0",
                 "productSemanticVersion": "17.1.0+32210.238",
                 "requiredEngineVersion": "3.1.2196.8931"
               },
               "properties": {
                 "campaignId": "2030",
                 "channelManifestId": "VisualStudio.17.Release/17.1.0+32210.238",
                 "nickname": "",
                 "setupEngineFilePath": "C:\\Program Files (x86)\\Microsoft Visual Studio\\Installer\\setup.exe"
               }
             }
           ]
[   +2 ms] Building Windows application...
[  +13 ms] <- compile package:win10/main.dart
[   +4 ms] executing: C:\Program Files\Microsoft Visual Studio\2022\Community\Common7\IDE\CommonExtensions\Microsoft\CMake\CMake\bin\cmake.exe -S
c:\Users\brian2\flutter-dev\testwin01\win10\windows -B build\windows -G Visual Studio 17 2022
[  +75 ms] -- Configuring done
[  +56 ms] -- Generating done
[   +4 ms] -- Build files have been written to: C:/Users/brian2/flutter-dev/testwin01/win10/build/windows
[   +7 ms] executing: C:\Program Files\Microsoft Visual Studio\2022\Community\Common7\IDE\CommonExtensions\Microsoft\CMake\CMake\bin\cmake.exe --build build\windows --config Debug --target
INSTALL --verbose
[ +145 ms] Microsoft (R) Build Engine version 17.1.0+ae57d105c for .NET Framework
[   +2 ms] Copyright (C) Microsoft Corporation. All rights reserved.
[  +84 ms] Build started 17/02/2022 11:21:30 PM.
[ +224 ms] Project "C:\Users\brian2\flutter-dev\testwin01\win10\build\windows\INSTALL.vcxproj" on node 1 (default targets).
[   +2 ms] Project "C:\Users\brian2\flutter-dev\testwin01\win10\build\windows\INSTALL.vcxproj" (1) is building "C:\Users\brian2\flutter-dev\testwin01\win10\build\windows\ZERO_CHECK.vcxproj"
(2) on node 1 (default targets).
[        ] InitializeBuildStatus:
[   +1 ms]   Creating "x64\Debug\ZERO_CHECK\ZERO_CHECK.tlog\unsuccessfulbuild" because "AlwaysCreate" was specified.
[  +18 ms] CustomBuild:
[   +1 ms]   All outputs are up-to-date.
[  +25 ms] FinalizeBuildStatus:
[        ]   Deleting file "x64\Debug\ZERO_CHECK\ZERO_CHECK.tlog\unsuccessfulbuild".
[   +2 ms]   Touching "x64\Debug\ZERO_CHECK\ZERO_CHECK.tlog\ZERO_CHECK.lastbuildstate".
[   +2 ms] Done Building Project "C:\Users\brian2\flutter-dev\testwin01\win10\build\windows\ZERO_CHECK.vcxproj" (default targets).
[  +82 ms] Project "C:\Users\brian2\flutter-dev\testwin01\win10\build\windows\INSTALL.vcxproj" (1) is building "C:\Users\brian2\flutter-dev\testwin01\win10\build\windows\ALL_BUILD.vcxproj"
(3) on node 1 (default targets).
[   +1 ms] Project "C:\Users\brian2\flutter-dev\testwin01\win10\build\windows\ALL_BUILD.vcxproj" (3) is building
"C:\Users\brian2\flutter-dev\testwin01\win10\build\windows\flutter\flutter_wrapper_app.vcxproj" (4) on node 1 (default targets).
[   +1 ms] Project "C:\Users\brian2\flutter-dev\testwin01\win10\build\windows\flutter\flutter_wrapper_app.vcxproj" (4) is building
"C:\Users\brian2\flutter-dev\testwin01\win10\build\windows\flutter\flutter_assemble.vcxproj" (5) on node 1 (default targets).
[   +2 ms] InitializeBuildStatus:
[        ]   Creating "x64\Debug\flutter_assemble\flutter_assemble.tlog\unsuccessfulbuild" because "AlwaysCreate" was specified.
[   +4 ms] CustomBuild:
[        ]   "The build of 'C:\Users\brian2\flutter-dev\testwin01\win10\build\windows\CMakeFiles\c9090866a9745b1f68178b1d642baa16\flutter_assemble.rule' depends on
'C:\USERS\BRIAN2\FLUTTER-DEV\TESTWIN01\WIN10\WINDOWS\FLUTTER\EPHEMERAL\FLUTTER_WINDOWS.DLL' which is produced by the build of
'C:\Users\brian2\flutter-dev\testwin01\win10\build\windows\CMakeFiles\26fe5c9e6d68e1e5df0fa4f0c74d911f\flutter_windows.dll.rule'. The items cannot be built in parallel."
[ +136 ms]   Generating C:/Users/brian2/flutter-dev/testwin01/win10/windows/flutter/ephemeral/flutter_windows.dll,
C:/Users/brian2/flutter-dev/testwin01/win10/windows/flutter/ephemeral/flutter_export.h, C:/Users/brian2/flutter-dev/testwin01/win10/windows/flutter/ephemeral/flutter_windows.h,
C:/Users/brian2/flutter-dev/testwin01/win10/windows/flutter/ephemeral/flutter_messenger.h, C:/Users/brian2/flutter-dev/testwin01/win10/windows/flutter/ephemeral/flutter_plugin_registrar.h,
C:/Users/brian2/flutter-dev/testwin01/win10/windows/flutter/ephemeral/flutter_texture_registrar.h,
C:/Users/brian2/flutter-dev/testwin01/win10/windows/flutter/ephemeral/cpp_client_wrapper/core_implementations.cc,
C:/Users/brian2/flutter-dev/testwin01/win10/windows/flutter/ephemeral/cpp_client_wrapper/standard_codec.cc,
C:/Users/brian2/flutter-dev/testwin01/win10/windows/flutter/ephemeral/cpp_client_wrapper/plugin_registrar.cc,
C:/Users/brian2/flutter-dev/testwin01/win10/windows/flutter/ephemeral/cpp_client_wrapper/flutter_engine.cc,
C:/Users/brian2/flutter-dev/testwin01/win10/windows/flutter/ephemeral/cpp_client_wrapper/flutter_view_controller.cc, _phony_
[ +843 ms]   [  +51 ms] executing: [c:\Users\brian2\flutter/] git -c log.showSignature=false log -n 1 --pretty=format:%H
[  +43 ms]   [  +61 ms] Exit code 0 from: git -c log.showSignature=false log -n 1 --pretty=format:%H
[   +1 ms]   [        ] db747aa1331bd95bc9b3874c842261ca2d302cd5
[   +2 ms]   [        ] executing: [c:\Users\brian2\flutter/] git tag --points-at db747aa1331bd95bc9b3874c842261ca2d302cd5
[  +33 ms]   [  +36 ms] Exit code 0 from: git tag --points-at db747aa1331bd95bc9b3874c842261ca2d302cd5
[   +2 ms]   [        ] 2.10.1
[   +5 ms]   [   +8 ms] executing: [c:\Users\brian2\flutter/] git rev-parse --abbrev-ref --symbolic @{u}
[  +27 ms]   [  +27 ms] Exit code 0 from: git rev-parse --abbrev-ref --symbolic @{u}
[        ]   [        ] origin/stable
[   +2 ms]   [        ] executing: [c:\Users\brian2\flutter/] git ls-remote --get-url origin
[  +22 ms]   [  +25 ms] Exit code 0 from: git ls-remote --get-url origin
[   +2 ms]   [        ] https://github.com/flutter/flutter.git
[  +46 ms]   [  +48 ms] executing: [c:\Users\brian2\flutter/] git rev-parse --abbrev-ref HEAD
[  +30 ms]   [  +30 ms] Exit code 0 from: git rev-parse --abbrev-ref HEAD
[   +1 ms]   [        ] stable
[  +39 ms]   [  +41 ms] Artifact Instance of 'AndroidGenSnapshotArtifacts' is not required, skipping update.
[   +2 ms]   [        ] Artifact Instance of 'AndroidInternalBuildArtifacts' is not required, skipping update.
[   +2 ms]   [        ] Artifact Instance of 'IOSEngineArtifacts' is not required, skipping update.
[        ]   [        ] Artifact Instance of 'FlutterWebSdk' is not required, skipping update.
[        ]   [   +3 ms] Artifact Instance of 'WindowsEngineArtifacts' is not required, skipping update.
[        ]   [        ] Artifact Instance of 'WindowsUwpEngineArtifacts' is not required, skipping update.
[        ]   [        ] Artifact Instance of 'MacOSEngineArtifacts' is not required, skipping update.
[        ]   [        ] Artifact Instance of 'LinuxEngineArtifacts' is not required, skipping update.
[        ]   [        ] Artifact Instance of 'LinuxFuchsiaSDKArtifacts' is not required, skipping update.
[        ]   [        ] Artifact Instance of 'MacOSFuchsiaSDKArtifacts' is not required, skipping update.
[        ]   [        ] Artifact Instance of 'FlutterRunnerSDKArtifacts' is not required, skipping update.
[        ]   [        ] Artifact Instance of 'FlutterRunnerDebugSymbols' is not required, skipping update.
[  +85 ms]   [  +89 ms] Artifact Instance of 'MaterialFonts' is not required, skipping update.
[   +2 ms]   [        ] Artifact Instance of 'GradleWrapper' is not required, skipping update.
[   +2 ms]   [        ] Artifact Instance of 'AndroidGenSnapshotArtifacts' is not required, skipping update.
[        ]   [        ] Artifact Instance of 'AndroidInternalBuildArtifacts' is not required, skipping update.
[        ]   [        ] Artifact Instance of 'IOSEngineArtifacts' is not required, skipping update.
[        ]   [        ] Artifact Instance of 'FlutterWebSdk' is not required, skipping update.
[        ]   [        ] Artifact Instance of 'FlutterSdk' is not required, skipping update.
[        ]   [   +2 ms] Artifact Instance of 'WindowsUwpEngineArtifacts' is not required, skipping update.
[        ]   [        ] Artifact Instance of 'MacOSEngineArtifacts' is not required, skipping update.
[        ]   [        ] Artifact Instance of 'LinuxEngineArtifacts' is not required, skipping update.
[        ]   [        ] Artifact Instance of 'LinuxFuchsiaSDKArtifacts' is not required, skipping update.
[        ]   [        ] Artifact Instance of 'MacOSFuchsiaSDKArtifacts' is not required, skipping update.
[        ]   [        ] Artifact Instance of 'FlutterRunnerSDKArtifacts' is not required, skipping update.
[        ]   [        ] Artifact Instance of 'FlutterRunnerDebugSymbols' is not required, skipping update.
[        ]   [        ] Artifact Instance of 'IosUsbArtifacts' is not required, skipping update.
[        ]   [        ] Artifact Instance of 'IosUsbArtifacts' is not required, skipping update.
[        ]   [        ] Artifact Instance of 'IosUsbArtifacts' is not required, skipping update.
[        ]   [        ] Artifact Instance of 'IosUsbArtifacts' is not required, skipping update.
[        ]   [        ] Artifact Instance of 'IosUsbArtifacts' is not required, skipping update.
[        ]   [        ] Artifact Instance of 'FontSubsetArtifacts' is not required, skipping update.
[        ]   [        ] Artifact Instance of 'PubDependencies' is not required, skipping update.
[  +27 ms]   [  +36 ms] Initializing file store
[  +13 ms]   [  +13 ms] Done initializing file store
[  +49 ms]   [  +48 ms] Skipping target: gen_localizations
[   +9 ms]   [   +9 ms] Skipping target: gen_dart_plugin_registrant
[+2322 ms]   [+2322 ms] Skipping target: unpack_windows
[ +481 ms]   [ +481 ms] Skipping target: kernel_snapshot
[ +160 ms]   [ +160 ms] Skipping target: aot_elf_release
[  +19 ms]   [  +19 ms] Skipping target: windows_aot_bundle
[  +37 ms]   [  +38 ms] Skipping target: release_bundle_windows_assets
[   +2 ms]   [        ] Persisting file store
[   +7 ms]   [   +9 ms] Done persisting file store
[   +3 ms]   [   +3 ms] build succeeded.
[   +7 ms]   [   +7 ms] "flutter assemble" took 3,258ms.
[ +270 ms]   [ +270 ms] ensureAnalyticsSent: 266ms
[   +2 ms]   [        ] Running shutdown hooks
[   +2 ms]   [        ] Shutdown hooks complete
[        ]   [        ] exiting with code 0
[  +73 ms] FinalizeBuildStatus:
[   +1 ms]   Deleting file "x64\Debug\flutter_assemble\flutter_assemble.tlog\unsuccessfulbuild".
[   +2 ms]   Touching "x64\Debug\flutter_assemble\flutter_assemble.tlog\flutter_assemble.lastbuildstate".
[        ] Done Building Project "C:\Users\brian2\flutter-dev\testwin01\win10\build\windows\flutter\flutter_assemble.vcxproj" (default targets).
[   +1 ms] InitializeBuildStatus:
[        ]   Creating "flutter_wrapper_app.dir\Debug\flutter_.F40F60E6.tlog\unsuccessfulbuild" because "AlwaysCreate" was specified.
[   +1 ms] CustomBuild:
[        ]   All outputs are up-to-date.
[  +55 ms] ClCompile:
[   +1 ms]   All outputs are up-to-date.
[  +22 ms] Lib:
[        ]   All outputs are up-to-date.
[   +2 ms]   flutter_wrapper_app.vcxproj -> C:\Users\brian2\flutter-dev\testwin01\win10\build\windows\flutter\Debug\flutter_wrapper_app.lib
[   +9 ms] FinalizeBuildStatus:
[        ]   Deleting file "flutter_wrapper_app.dir\Debug\flutter_.F40F60E6.tlog\unsuccessfulbuild".
[   +2 ms]   Touching "flutter_wrapper_app.dir\Debug\flutter_.F40F60E6.tlog\flutter_wrapper_app.lastbuildstate".
[        ] Done Building Project "C:\Users\brian2\flutter-dev\testwin01\win10\build\windows\flutter\flutter_wrapper_app.vcxproj" (default targets).
[  +19 ms] Project "C:\Users\brian2\flutter-dev\testwin01\win10\build\windows\ALL_BUILD.vcxproj" (3) is building
"C:\Users\brian2\flutter-dev\testwin01\win10\build\windows\flutter\flutter_wrapper_plugin.vcxproj" (6) on node 1 (default targets).
[   +1 ms] InitializeBuildStatus:
[   +3 ms]   Creating "flutter_wrapper_plugin.dir\Debug\flutter_.1217592C.tlog\unsuccessfulbuild" because "AlwaysCreate" was specified.
[        ] CustomBuild:
[   +3 ms]   All outputs are up-to-date.
[   +1 ms] ClCompile:
[        ]   All outputs are up-to-date.
[   +3 ms] Lib:
[        ]   All outputs are up-to-date.
[        ]   flutter_wrapper_plugin.vcxproj -> C:\Users\brian2\flutter-dev\testwin01\win10\build\windows\flutter\Debug\flutter_wrapper_plugin.lib
[   +4 ms] FinalizeBuildStatus:
[   +1 ms]   Deleting file "flutter_wrapper_plugin.dir\Debug\flutter_.1217592C.tlog\unsuccessfulbuild".
[   +1 ms]   Touching "flutter_wrapper_plugin.dir\Debug\flutter_.1217592C.tlog\flutter_wrapper_plugin.lastbuildstate".
[        ] Done Building Project "C:\Users\brian2\flutter-dev\testwin01\win10\build\windows\flutter\flutter_wrapper_plugin.vcxproj" (default targets).
[  +23 ms] Project "C:\Users\brian2\flutter-dev\testwin01\win10\build\windows\ALL_BUILD.vcxproj" (3) is building
"C:\Users\brian2\flutter-dev\testwin01\win10\build\windows\runner\win10.vcxproj" (7) on node 1 (default targets).
[        ] InitializeBuildStatus:
[   +3 ms]   Creating "win10.dir\Debug\win10.tlog\unsuccessfulbuild" because "AlwaysCreate" was specified.
[        ] CustomBuild:
[   +2 ms]   All outputs are up-to-date.
[  +11 ms] ClCompile:
[        ]   C:\Program Files\Microsoft Visual Studio\2022\Community\VC\Tools\MSVC\14.31.31103\bin\HostX64\x64\CL.exe /c /I"C:\Users\brian2\flutter-dev\testwin01\win10\windows"
/I"C:\Users\brian2\flutter-dev\testwin01\win10\windows\flutter\ephemeral" /I"C:\Users\brian2\flutter-dev\testwin01\win10\windows\flutter\ephemeral\cpp_client_wrapper\include" /Zi /nologo /W4
/WX /diagnostics:column /Od /Ob0 /D _UNICODE /D UNICODE /D WIN32 /D _WINDOWS /D _HAS_EXCEPTIONS=0 /D _DEBUG /D NOMINMAX /D UNICODE /D _UNICODE /D "CMAKE_INTDIR=\"Debug\"" /Gm- /EHsc /RTC1
/MDd /GS /fp:precise /Zc:wchar_t /Zc:forScope /Zc:inline /GR /std:c++17 /Fo"win10.dir\Debug\\" /Fd"win10.dir\Debug\vc143.pdb" /external:W4 /Gd /TP /wd4100 /errorReport:queue
"C:\Users\brian2\flutter-dev\testwin01\win10\windows\flutter\generated_plugin_registrant.cc" "C:\Users\brian2\flutter-dev\testwin01\win10\windows\runner\flutter_window.cpp"
[  +54 ms]   generated_plugin_registrant.cc
[ +429 ms]   flutter_window.cpp
[ +757 ms]   Generating Code...
[  +62 ms] ResourceCompile:
[   +1 ms]   All outputs are up-to-date.
[  +28 ms] Link:
[   +1 ms]   C:\Program Files\Microsoft Visual Studio\2022\Community\VC\Tools\MSVC\14.31.31103\bin\HostX64\x64\link.exe /ERRORREPORT:QUEUE
/OUT:"C:\Users\brian2\flutter-dev\testwin01\win10\build\windows\runner\Debug\win10.exe" /INCREMENTAL /ILK:"win10.dir\Debug\win10.ilk" /NOLOGO ..\flutter\Debug\flutter_wrapper_app.lib
"C:\Users\brian2\flutter-dev\testwin01\win10\windows\flutter\ephemeral\flutter_windows.dll.lib" kernel32.lib user32.lib gdi32.lib winspool.lib shell32.lib ole32.lib oleaut32.lib uuid.lib
comdlg32.lib advapi32.lib /MANIFEST /MANIFESTUAC:"level='asInvoker' uiAccess='false'" /manifest:embed
/manifestinput:"C:\Users\brian2\flutter-dev\testwin01\win10\windows\runner\runner.exe.manifest" /DEBUG /PDB:"C:/Users/brian2/flutter-dev/testwin01/win10/build/windows/runner/Debug/win10.pdb"
/SUBSYSTEM:WINDOWS /TLBID:1 /DYNAMICBASE /NXCOMPAT /IMPLIB:"C:/Users/brian2/flutter-dev/testwin01/win10/build/windows/runner/Debug/win10.lib" /MACHINE:X64  /machine:x64
win10.dir\Debug\Runner.res
[   +2 ms]   win10.dir\Debug\flutter_window.obj
[   +1 ms]   win10.dir\Debug\main.obj
[        ]   win10.dir\Debug\utils.obj
[        ]   win10.dir\Debug\win32_window.obj
[        ]   win10.dir\Debug\generated_plugin_registrant.obj
[ +404 ms]   win10.vcxproj -> C:\Users\brian2\flutter-dev\testwin01\win10\build\windows\runner\Debug\win10.exe
[   +8 ms] FinalizeBuildStatus:
[   +1 ms]   Deleting file "win10.dir\Debug\win10.tlog\unsuccessfulbuild".
[        ]   Touching "win10.dir\Debug\win10.tlog\win10.lastbuildstate".
[        ] Done Building Project "C:\Users\brian2\flutter-dev\testwin01\win10\build\windows\runner\win10.vcxproj" (default targets).
[   +6 ms] InitializeBuildStatus:
[   +1 ms]   Creating "x64\Debug\ALL_BUILD\ALL_BUILD.tlog\unsuccessfulbuild" because "AlwaysCreate" was specified.
[   +1 ms] CustomBuild:
[        ]   All outputs are up-to-date.
[  +14 ms] FinalizeBuildStatus:
[   +1 ms]   Deleting file "x64\Debug\ALL_BUILD\ALL_BUILD.tlog\unsuccessfulbuild".
[        ]   Touching "x64\Debug\ALL_BUILD\ALL_BUILD.tlog\ALL_BUILD.lastbuildstate".
[        ] Done Building Project "C:\Users\brian2\flutter-dev\testwin01\win10\build\windows\ALL_BUILD.vcxproj" (default targets).
[   +4 ms] InitializeBuildStatus:
[   +1 ms]   Creating "x64\Debug\INSTALL\INSTALL.tlog\unsuccessfulbuild" because "AlwaysCreate" was specified.
[  +56 ms] PostBuildEvent:
[   +2 ms]   setlocal
[   +2 ms]   "C:\Program Files\Microsoft Visual Studio\2022\Community\Common7\IDE\CommonExtensions\Microsoft\CMake\CMake\bin\cmake.exe" -DBUILD_TYPE=Debug -P cmake_install.cmake
[        ]   if %errorlevel% neq 0 goto :cmEnd
[        ]   :cmEnd
[        ]   endlocal & call :cmErrorLevel %errorlevel% & goto :cmDone
[        ]   :cmErrorLevel
[        ]   exit /b %1
[        ]   :cmDone
[        ]   if %errorlevel% neq 0 goto :VCEnd
[        ]   :VCEnd
[  +43 ms]   -- Install configuration: "Debug"
[   +2 ms]   -- Up-to-date: C:/Users/brian2/flutter-dev/testwin01/win10/build/windows/runner/Debug/data/icudtl.dat
[        ]   -- Up-to-date: C:/Users/brian2/flutter-dev/testwin01/win10/build/windows/runner/Debug/flutter_windows.dll
[   +3 ms]   -- Installing: C:/Users/brian2/flutter-dev/testwin01/win10/build/windows/runner/Debug/data/flutter_assets
[        ]   -- Installing: C:/Users/brian2/flutter-dev/testwin01/win10/build/windows/runner/Debug/data/flutter_assets/AssetManifest.json
[        ]   -- Installing: C:/Users/brian2/flutter-dev/testwin01/win10/build/windows/runner/Debug/data/flutter_assets/FontManifest.json
[        ]   -- Installing: C:/Users/brian2/flutter-dev/testwin01/win10/build/windows/runner/Debug/data/flutter_assets/fonts
[   +1 ms]   -- Installing: C:/Users/brian2/flutter-dev/testwin01/win10/build/windows/runner/Debug/data/flutter_assets/fonts/MaterialIcons-Regular.otf
[   +3 ms]   -- Installing: C:/Users/brian2/flutter-dev/testwin01/win10/build/windows/runner/Debug/data/flutter_assets/NOTICES.Z
[   +1 ms]   -- Installing: C:/Users/brian2/flutter-dev/testwin01/win10/build/windows/runner/Debug/data/flutter_assets/packages
[        ]   -- Installing: C:/Users/brian2/flutter-dev/testwin01/win10/build/windows/runner/Debug/data/flutter_assets/packages/cupertino_icons
[        ]   -- Installing: C:/Users/brian2/flutter-dev/testwin01/win10/build/windows/runner/Debug/data/flutter_assets/packages/cupertino_icons/assets
[        ]   -- Installing: C:/Users/brian2/flutter-dev/testwin01/win10/build/windows/runner/Debug/data/flutter_assets/packages/cupertino_icons/assets/CupertinoIcons.ttf
[   +9 ms] FinalizeBuildStatus:
[   +4 ms]   Deleting file "x64\Debug\INSTALL\INSTALL.tlog\unsuccessfulbuild".
[        ]   Touching "x64\Debug\INSTALL\INSTALL.tlog\INSTALL.lastbuildstate".
[        ] Done Building Project "C:\Users\brian2\flutter-dev\testwin01\win10\build\windows\INSTALL.vcxproj" (default targets).
[        ] Build succeeded.
[        ]     0 Warning(s)
[        ]     0 Error(s)
[        ] Time Elapsed 00:00:07.33
[  +21 ms] Building Windows application... (completed in 7.8s)
[  +97 ms] Can't load AOT data from c:\Users\brian2\flutter-dev\testwin01\win10\build\windows\runner\Debug\data\app.so; no such file.
[   +2 ms] Unable to start engine without AOT data.
[   +9 ms] Failed to create view controller.
[  +73 ms] Error waiting for a debug connection: The log reader stopped unexpectedly, or never started.
[   +3 ms] Error launching application on Windows.
[   +5 ms] "flutter run" took 8,347ms.
[   +3 ms]
           #0      throwToolExit (package:flutter_tools/src/base/common.dart:10:3)
           #1      RunCommand.runCommand (package:flutter_tools/src/commands/run.dart:687:9)
           <asynchronous suspension>
           #2      FlutterCommand.run.<anonymous closure> (package:flutter_tools/src/runner/flutter_command.dart:1161:27)
           <asynchronous suspension>
           #3      AppContext.run.<anonymous closure> (package:flutter_tools/src/base/context.dart:150:19)
           <asynchronous suspension>
           #4      CommandRunner.runCommand (package:args/command_runner.dart:209:13)
           <asynchronous suspension>
           #5      FlutterCommandRunner.runCommand.<anonymous closure> (package:flutter_tools/src/runner/flutter_command_runner.dart:281:9)
           <asynchronous suspension>
           #6      AppContext.run.<anonymous closure> (package:flutter_tools/src/base/context.dart:150:19)
           <asynchronous suspension>
           #7      FlutterCommandRunner.runCommand (package:flutter_tools/src/runner/flutter_command_runner.dart:229:5)
           <asynchronous suspension>
           #8      run.<anonymous closure>.<anonymous closure> (package:flutter_tools/runner.dart:62:9)
           <asynchronous suspension>
           #9      AppContext.run.<anonymous closure> (package:flutter_tools/src/base/context.dart:150:19)
           <asynchronous suspension>
           #10     main (package:flutter_tools/executable.dart:94:3)
           <asynchronous suspension>


[ +146 ms] ensureAnalyticsSent: 144ms
[   +2 ms] Running shutdown hooks
[   +2 ms] Shutdown hooks complete
[   +1 ms] exiting with code 1

c:\Users\brian2\flutter-dev\testwin01\win10>

[compile-log.pdf](https://github.com/flutter/flutter/files/8089903/compile-log.pdf)


****** ATTEMPT TO CREATE WINDOWS VERSION ****

c:\Users\brian2\flutter-dev\testwin01>flutter create --platforms=windows .\win10
Creating project win10...
Running "flutter pub get" in win10...                            2,009ms
Wrote 28 files.

All done!
In order to run your application, type:

  $ cd win10
  $ flutter run

Your application code is in win10\lib\main.dart.


******* ATTEMPT TO RUN THE WINDOWS VERSION ****

c:\Users\brian2\flutter-dev\testwin01>cd win10

c:\Users\brian2\flutter-dev\testwin01\win10>flutter run
Launching lib\main.dart on Windows in debug mode...
Building Windows application...
Can't load AOT data from c:\Users\brian2\flutter-dev\testwin01\win10\build\windows\runner\Debug\data\app.so; no such file.
Unable to start engine without AOT data.
Failed to create view controller.
Error waiting for a debug connection: The log reader stopped unexpectedly, or never started.
Error launching application on Windows.

c:\Users\brian2\flutter-dev\testwin01\win10>

c:\Users\brian2\flutter-dev\testwin01\win10>flutter --version
Flutter 2.10.1 • channel stable • https://github.com/flutter/flutter.git
Framework • revision db747aa133 (8 days ago) • 2022-02-09 13:57:35 -0600
Engine • revision ab46186b24
Tools • Dart 2.16.1 • DevTools 2.9.2

********  ERROR ABOVE ATTEMPTING TO RUN WINDOWS VERSION ***
```

```
****** RUN FLUTTER DOCTOR ****

c:\Users\brian2\flutter-dev>cd testwin01

c:\Users\brian2\flutter-dev\testwin01>flutter doctor -v
[√] Flutter (Channel stable, 2.10.1, on Microsoft Windows [Version 10.0.22000.493], locale en-AU)
    • Flutter version 2.10.1 at c:\Users\brian2\flutter
    • Upstream repository https://github.com/flutter/flutter.git
    • Framework revision db747aa133 (8 days ago), 2022-02-09 13:57:35 -0600
    • Engine revision ab46186b24
    • Dart version 2.16.1
    • DevTools version 2.9.2

[√] Android toolchain - develop for Android devices (Android SDK version 30.0.3)
    • Android SDK at C:\Users\brian2\AppData\Local\Android\sdk
    • Platform android-30, build-tools 30.0.3
    • Java binary at: C:\Program Files\Android\Android Studio\jre\bin\java
    • Java version OpenJDK Runtime Environment (build 1.8.0_242-release-1644-b01)
    • All Android licenses accepted.

[√] Chrome - develop for the web
    • Chrome at C:\Program Files\Google\Chrome\Application\chrome.exe

[√] Visual Studio - develop for Windows (Visual Studio Community 2022 17.1.0)
    • Visual Studio at C:\Program Files\Microsoft Visual Studio\2022\Community
    • Visual Studio Community 2022 version 17.1.32210.238
    • Windows 10 SDK version 10.0.22000.0

[√] Android Studio (version 4.1)
    • Android Studio at C:\Program Files\Android\Android Studio
    • Flutter plugin can be installed from:
       https://plugins.jetbrains.com/plugin/9212-flutter
    • Dart plugin can be installed from:
       https://plugins.jetbrains.com/plugin/6351-dart
    • Java version OpenJDK Runtime Environment (build 1.8.0_242-release-1644-b01)

[√] VS Code (version 1.59.1)
    • VS Code at C:\Users\brian2\AppData\Local\Programs\Microsoft VS Code
    • Flutter extension can be installed from:
       https://marketplace.visualstudio.com/items?itemName=Dart-Code.flutter

[√] Connected device (3 available)
    • Windows (desktop) • windows • windows-x64    • Microsoft Windows [Version 10.0.22000.493]
    • Chrome (web)      • chrome  • web-javascript • Google Chrome 96.0.4664.45
    • Edge (web)        • edge    • web-javascript • Microsoft Edge 98.0.1108.50

[√] HTTP Host Availability
    • All required HTTP hosts are available

• No issues found!
```

</details>

**Comment by stuartmorgan** [comment]
Duplicate of https://github.com/flutter/flutter/issues/97086
